packages/connectivity/src/scp-cf/destination/destination-from-register.ts

```diff
@@ -0,0 +1,140 @@
+import { createLogger } from '@sap-cloud-sdk/util';
```

Comment:
Minor: The file name "from register" is a bit weird. I am thinking of a cash register, when reading that. Consider "from registration" - but we can also change that later - maybe "from" is not a good idea for this.